### PR TITLE
Added Pima Schema. 

### PR DESCRIPTION
**Pull request by jimduff-usds**
This PR now includes generation of data for Pima County.  

## Changes
- Tweaked Element so that it processes an element with a set of local altValues but no Value Set.
- Added Pima Schema, 95% same as input PDI schema.

## Checklist
- [X] Tested locally?
- [N] Added new dependencies?
- [ ] Updated the docs?
- [ ] Added a test?
- [X] Did you check for sensitive data, and remove any?
- [ ] Are additional approvals needed for this change?
- [ ] Are there potential vulnerabilities or licensing issues with any new dependencies introduced?

## Fixes 

See above change to Element.kt

## To Be Done

- Need to get the rest of the Devices from James.

### CODE REVIEW

File: prime-router/metadata/schemas/PrimeDataInput/pdi-covid-19.schema [hunk 1]
```diff
@@ -16,6 +16,48 @@ elements:
   - name: standard.patient_suffix
     csvFields: [{name: Patient_suffix}]
 
+  # Patient ID is generated internally by SimpleReport
```
Comment (jimduff-usds) [inline prime-router/metadata/schemas/PrimeDataInput/pdi-covid-19.schema:19]: Reordered these fields to make them more consistent.

File: prime-router/metadata/organizations.yml [hunk 2]
```diff
@@ -33,7 +33,7 @@
       - name: elr
         topic: covid-19
         schema: az/pima-az-covid-19
-        jurisdictionalFilter: { standard.testing_lab_state: AZ, standard.patient_county: Pima }
+        jurisdictionalFilter: { standard.testing_lab_state: AZ, standard.ordering_facility_county: "(?i).*Pima.*" }
         transforms: { deidentify: false }
```
Comment (RickHawesUSDS) [inline prime-router/metadata/organizations.yml:37]: nice.

File: prime-router/src/main/kotlin/Element.kt [hunk 3]
```diff
@@ -140,17 +140,21 @@ data class Element(
...
-                val set = Metadata.findValueSet(valueSet)
-                    ?: error("Schema Error: invalid valueSet name: $valueSet")
-                // TODO: A more flexible form of the format field for codes is possible and necessary
-                when (field?.format) {
-                    displayFormat -> set.toDisplayFromCode(normalizedValue)
-                        ?: error("Internal Error: '$normalizedValue' cannot be formatted for '$name'")
-                    altDisplayFormat -> toAltDisplay(normalizedValue)
+                // First, prioritize use of a local $alt format, even if no value set exists.
+                if (field?.format == altDisplayFormat) {
+                    toAltDisplay(normalizedValue)
```
Comment (RickHawesUSDS) [inline prime-router/src/main/kotlin/Element.kt:145]: Looks like a good improvement.

**Review by RickHawesUSDS — APPROVED** [review]
I approve, but please look at @jlusds comments about missing fields.

**Comment by jimduff-usds** [comment]
Just to document @jlusds comments, which says these fields are missing:

Order_test_date - It looks like there are three dates expected (Order_test_date = Time of Test, Test_date = Test result date, Date_result_released = Test report date). The latter two are included.
Role - This is a field like Staff, Patient, Visitor, Student, etc.
First_symptom - this is supposed to be Illness_onset_date

**Pull request merged**